### ISSUE TITLE
[Sorta Issue] Overlapping regions, showing titles to newly entered (lower priority) region.

### ISSUE THREAD

**Issue by AZFire1** [EngineHub/WorldGuard#1638]
**Versions**

https://paste.helpch.at/ebimoloror.cs

**Describe the bug**
You have multiple overlapping regions, you have titles setup in each of them. There is one region that is overlapping in 2 other regions and you enter that region while being one of the 2 other regions and move to the second region (out of the two regions). I have the main region that I am in which has the highest priority but the region I'm entering still sends the greeting-title's for it's region although the region I'm in has the highest priority and should override that. Right? I may have it wrong, but I think it works like that.

**To Reproduce**
1: Make a region.
2: Make 2 regions that are inside that first region (but don't overlap eachother, but are like 1 block apart)
3: Set priority for the first region higher than the others.
4: Set titles for each of the regions.
5: Switch between the 2 (low priority) regions while in the first (main) region and see the titles still play.

**Expected behavior**
If you switch between the regions while in the highest priority region and you have titles for all 3 it will show you the titles for the low priority regions anyways.

**Additional context**
None

**Comment by me4502** [comment]
Is this the same as https://github.com/EngineHub/WorldGuard/issues/1637 ?

Please don't make duplicated issues

**Comment by AZFire1** [comment]
No, it has to do with titles though I wouldn't consider it the same.
